Libs/Mesh/Mesh.cpp

```diff
@@ -1038,6 +1096,411 @@ std::ostream& operator<<(std::ostream &os, const Mesh& mesh)
   return os;
 }
 
+//Splits the mesh for FFCs by setting scalar and vector fields
```

Comment:
nit-picking, but coding standards ask that documentation should be a triple-slash-prefixed comment in the header. That way it is reflected consistently and within Doxygen API. Please go through the set of functions and make it consistent with the rest of the class. Thanks!

Reply:
Done